GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -738,136 +742,185 @@ - (void)testNotHandleWrongPath {
 
 #pragma mark - Tests - disconnectWithCallback:
 
+// Verifies the request url when disconnect succeeds.
+- (void)testDisconnect_verifyURL_succeed {
+  [_keychainHandler saveAuthState:_authState];
+  OCMStub([_authState lastTokenResponse]).andReturn(_tokenResponse);
+  OCMStub([_tokenResponse accessToken]).andReturn(kAccessToken);
+  
+  XCTestExpectation *expectation1 =
+      [self expectationWithDescription:@"test block is called"];
+  GIDHTTPFetcherTestBlock testBlock =
+      ^(NSURLRequest *request, GIDHTTPFetcherFakeResponseHandlerBlock response) {
+        NSURL *url = request.URL;
+        XCTAssertEqualObjects([url scheme], @"https", @"scheme must match");
+        XCTAssertEqualObjects([url host], @"accounts.google.com", @"host must match");
+        XCTAssertEqualObjects([url path], @"/o/oauth2/revoke", @"path must match");
+        OIDURLQueryComponent *queryComponent = [[OIDURLQueryComponent alloc] initWithURL:url];
+        NSDictionary<NSString *, NSObject<NSCopying> *> *params = queryComponent.dictionaryValue;
+        XCTAssertEqualObjects([params valueForKey:kSDKVersionLoggingParameter], GIDVersion(),
+                              @"SDK version logging parameter should match");
+        XCTAssertEqualObjects([params valueForKey:kEnvironmentLoggingParameter], GIDEnvironment(),
+                              @"Environment logging parameter should match");
+        
+        NSData *body = request.HTTPBody;
+        NSString* bodyString = [[NSString alloc] initWithData:body encoding:NSUTF8StringEncoding];
+        NSArray<NSString *> *strings = [bodyString componentsSeparatedByString:@"="];
+        XCTAssertEqualObjects(strings[1], kAccessToken);
+
+        NSData *data = [[NSData alloc] init];
+        response(data, nil);
+        [expectation1 fulfill];
+     };
+  [_httpFetcher setTestBlock:testBlock];
+  
+  XCTestExpectation *expectation =
```

Comment:
Since there's more than one expectation in this case, use a more descriptive name, like `completionExpectation`.